wiki/Storyboarding/id.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Storyboarding
+
+<!-- TODO: Needs to fit in with storyboard scripting somehow -->
+
+<!-- Two important links are wiki/Beatmap_Editor/Design (visual) and wiki/Storyboard_Scripting (manual coding done by KJ) -->
+
+## Membuat sebuah storyboard
+
+[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (shortcut keyboard F2) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap sebagai kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard untuk layar lebar (widescreen), Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
```

Comment:
```suggestion
[Storyboard Editor](/wiki/Beatmap_Editor/Design) (SBE) terletak pada Tab Desain (dapat diakses dengan shortcut F2 pada keyboard) di dalam Beatmap Editor. Pada dasarnya, SBE memberikan pengguna cara yang mudah untuk mengontrol pergerakan sprite, tetapi hal itu selalu dianggap kaku dan tidak berguna untuk semua hal yang lebih dari dasar storyboarding. Jika Anda ingin membuat storyboard untuk layar lebar (widescreen), Anda dapat melakukannya melalui dialog pengaturan lagu dan mengecek kotak Widescreen Support pada Tab Desain.
```

Reply:
Damn I double-type that `sebagai` hahaha